### ISSUE TITLE
[Raid] Heroic and Normal should share ID !

### ISSUE THREAD

**Issue by Slydee23** [WoWManiaUK/Blackwing-Lair#3615]
**Links:**
https://wow.gamepedia.com/Raid_lockout#Cataclysm_raids

**What is happening:**

at the moment you can Do Heroic and NOrmal Raid in the Same ID,

**What should happen:**

You should only be able to do the Raid once per week doesnt matter if Normal or Heroic

**Comment by Slydee23** [comment]
ruins the whole Raid Progress on the Server.

**Comment by Rushor** [comment]
e1fc0697fb08860dbde59a874c91b5ecfecf9d4f

**Comment by Rushor** [comment]
i will fix that with some conditions. you first need to complete the endboss in nh already of the instance.

all player nned to be outside of the raid in order to switch the raid form.

**Comment by Slydee23** [comment]
yeah thats right you need to kill the Endboss once  in Normal as raidlead otherwise nobody should be able to enter Heroic mode.

**Comment by Rushor** [comment]
could you inspect that @keader i can give you a plain fix, but it needs to be improved :o

**Comment by questbugreporter** [comment]
There's more: I just logged on a character that had only killed valiona and theralion with the same group that continued on to kill cho'gall and all the bosses were dead.
Our characters seem to be locked to the specific instance and not just to the bosses that get killed while you're in the group.

**Comment by Rushor** [comment]
eh that's how it always works? you get saved to the instance, not to the individual char progress?

**Comment by questbugreporter** [comment]
How to replicate: 

Get 4 characters from 2 different accounts: A, B, C and D
Kill Halfus with characters A and B, then kill twin dragons with characters  B and C.
Then log on characters A and D and check what bosses are still alive.

**Comment by questbugreporter** [comment]
Maybe that's how it worked in wotlk (which I've never played.)
Cataclysm+ doesn't save you to a specific instance.

**Comment by Rushor** [comment]
can you show me some more details reports about that? i don't think we have this mechanic at all currently

**Comment by questbugreporter** [comment]
That's gonna be hard as the system has changed again since then.
In legion you could kill bosses any number of times but you wouldn't get loot.
I'm going to try to find proof though.

**Comment by APTX4869s** [comment]
![Locked](https://user-images.githubusercontent.com/57665529/70094941-00c5ca80-163d-11ea-94d4-cfc931a3d137.png)
this what it says 
I killed Halfus and the dragons in another group ( I stopped at countil ), when I wanted to join another raid group that stopped at council it shows that.
I should be able to join them normally because we have the same id
( this relates to issue 2 which it got reported by @questbugreporter  )

**Comment by questbugreporter** [comment]
@Rushor https://www.wowhead.com/news=169667/4-0-1-new-flexible-raid-lockout-system

Wasn't hard after all.

**Comment by Echinaceax** [comment]
@Rushor 

![cata](https://user-images.githubusercontent.com/39439201/70095875-154a9880-1626-11ea-9905-00933125483b.jpg)

**Comment by Echinaceax** [comment]
update also tested and confirmed for Blackwing descent
